### ISSUE TITLE
Change location of unit tests

### ISSUE THREAD

**Issue by timofeysie** [Swiip/generator-gulp-angular#458]
I want to put the unit tests outside the project source directory.  I have a folder in the projects root directory where I want to keep the tests.  In gulp/unit-tests.js I change the testFiles variable like this:
```
  var testFiles = bowerDeps.js.concat([
    paths.src + 'test/**/*.js'
  ]);
```
But the tests can't be found.  I also added the test directory in the karma.conf.js file to no avail.
What do I need to do to change the location of the unit tests?

Thanks in advance.

**Comment by zckrs** [comment]
>folder in the projects root directory 

You should remove `paths.src` in 
`paths.src + 'test/**/*.js'`

**Comment by timofeysie** [comment]
Thanks for that.  The tests are running now.  However, they are failing to load modules and I have noticed that the karma.conf.js file also has what looks like conflicting configuration info in the files property.  Should the karma.conf.js files be set the same as the unit-tests.js testFiles?
Also, it seems like bower_components/**/*.js needs to have a leading /, but the test folder not.  What is the deal with that?

Thanks for your help.

**Comment by Swiip** [comment]
It should be easier when this will be merged  #499

**Comment by zckrs** [comment]
:tada: Just released v0.12.0 :tada:
Your issue should be fixed. I close but if you can retry your case with a new fresh install of generator you are welcome. Any feedback would be appreciated (positive comments too ^_^).
